src/sentry/utils/samples.py

```diff
@@ -8,11 +8,20 @@
 from __future__ import absolute_import
 
 import os.path
+from datetime import datetime, timedelta
 
 from sentry.constants import DATA_ROOT
 from sentry.event_manager import EventManager
 from sentry.utils import json
 
+epoch = datetime.utcfromtimestamp(0)
+now = datetime.now()
+
+
+def milliseconds_ago(milliseconds):
```

Comment:
I'm sure there's a better way to do this ...

Tricky thing is we don't have the `Event` date inside `load_data`, so I figured just offsetting from `datetime.now()` is good enough (if not entirely accurate since `load_data` is called before `Event` creation).